lib/jxl/dec_frame.h

```diff
@@ -104,6 +104,11 @@ class FrameDecoder {
   // Must be called exactly once per frame, after all calls to ProcessSections.
   Status FinalizeFrame();
 
+  // Returns dependencies of this frame as a bit mask: bits 0-3 indicate
+  // reference frame 0-3 for patches and blending, bits 4-7 indicate DC
+  // frames this frame depends on. Must be called
+  int GetDependencies() const;
```

Comment:
this is likely unintentional / from another branch, right?

Reply:
oops, removed from here